packages/kolibri-sandbox/webpack.config.h5p.js

```diff
@@ -31,7 +31,7 @@ module.exports = {
     filename: 'h5p-[contenthash].js',
     path: path.resolve(__dirname, '../../kolibri/core/content/static/h5p'),
   },
-  mode: 'none',
+  mode: 'production',
```

Comment:
No idea why way back when I decided to do this - probably because I copied it from the main kolibri-sandbox webpack configuration.